bmq-sdk/src/main/java/com/bloomberg/bmq/impl/infr/util/expressionvalidator/ValidationResult.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Bloomberg Finance L.P.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.bloomberg.bmq.impl.infr.util.expressionvalidator;
+
+public class ValidationResult {
+    private String errorMessage;
+    private boolean success;
```

Comment:
Let's make these fields final and mark this class as immutable, this might help:

```
import javax.annotation.concurrent.Immutable;
@Immutable
```

Reply:
Marked class as `immutable`